packages/mutation-testing-report-schema/src/api/MutantResult.ts

```diff
@@ -3,7 +3,8 @@ import { Location } from './Location';
 export interface MutantResult {
   id: string;
   mutatorName: string;
-  replacement: string;
+  replacement?: string;
```

Comment:
I'm still missing the changes in the schema and the unit tests to verify the schema.